stats/src/main/java/com/facebook/stats/mx/JVMStatsExporter.java

```diff
@@ -49,6 +54,34 @@
     .add(SimpleType.DOUBLE)
     .build();
 
+  private static final Set<String> NUMERIC_STRING_TYPES = ImmutableSet.<String>builder()
```

Comment:
Messenger code had it, so I though i should add as well. We can delete it for now and see if we need to add it later in another diff